.changeset/spicy-vans-add.md

```diff
@@ -0,0 +1,8 @@
+---
+'@finos/legend-application': patch
+'@finos/legend-application-query': patch
+'@finos/legend-application-studio': patch
+'@finos/legend-application-taxonomy': patch
+'@finos/legend-query-builder': patch
+---
+Enable virtual assistant for all applications ([#1712](https://github.com/finos/legend-studio/issues/1712)).
```

Comment:
This changeset should only apply for `@finos/legend-application` and it should be a `minor` change